src/tests/unit/tests/injected/components/command-bar.test.tsx

```diff
@@ -139,13 +151,11 @@ describe('CommandBar', () => {
                 onClickCopyIssueDetailsButton: onClickMock.object,
             };
 
-            const wrapper = shallow(<CommandBar {...props} />);
-
-            const button = wrapper.find(CopyIssueDetailsButton);
+            render(<CommandBar {...props} />);
 
-            button.prop('onClick')(eventStub);
+            getMockComponentClassPropsForCall(CopyIssueDetailsButton).onClick();
 
-            onClickMock.verify(onClick => onClick(eventStub), Times.once());
+            onClickMock.setup(onClick => onClick(eventStub)).verifiable(Times.once());
```

Comment:
This can stay the same as it was before.
```suggestion
            onClickMock.verify(onClick => onClick(eventStub), Times.once());
```